iocore/cache/Cache.cc

```diff
@@ -2754,14 +2759,36 @@ cplist_reconfigure()
           Warning("no volumes created");
           return -1;
         }
-        int64_t space_in_blks = static_cast<int64_t>(((config_vol->percent / percent_remaining)) * tot_space_in_blks);
+
+        // Find if the volume is forced and if it is then calculate the total forced volume size.
+        // Forced volumes take the whole span (disk) also sum all disk space this volume is forced to.
+        bool forcedVolume                = false;
```

Comment:
Why bother with this `bool`? Isn't `tot_forced_space_in_blks == 0` a reasonable substitute?

Reply:
OK, fixed. Although I thought it is more human-readable with the `bool` (also it is a stack variable in a non-performance-critical path, possibly also will be optimized out).